ibis/backends/tests/test_array.py

```diff
@@ -423,17 +427,21 @@ def test_array_slice(backend, start, stop):
         param({"a": [[1, 2], [4]]}, {"a": [[2, 3], [5]]}, id="no_nulls"),
     ],
 )
-def test_array_map(backend, con, input, output):
+def test_array_map(con, input, output):
     t = ibis.memtable(input, schema=ibis.schema(dict(a="!array<int8>")))
     expected = pd.DataFrame(output)
 
     expr = t.select(a=t.a.map(lambda x: x + 1))
     result = con.execute(expr)
-    backend.assert_frame_equal(result, expected)
+    assert frozenset(map(frozenset, result["a"])) == frozenset(
+        map(frozenset, expected["a"])
+    )
 
     expr = t.select(a=t.a.map(functools.partial(lambda x, y: x + y, y=1)))
     result = con.execute(expr)
-    backend.assert_frame_equal(result, expected)
+    assert frozenset(map(frozenset, result["a"])) == frozenset(
```

Comment:
A lot has changed in this test file. Apparently now you are discarding the order of the array elements, why is that?

If only trino needs that could we keep to ordered comparison for the other backends?

Reply:
It should be only the order of the rows that's discarded, let me make sure that's what's happening.

Reply:
Confirmed the converting the inner `frozenset` to `tuple` works fine. Pushing now.